Documentation/LadybirdBuildInstructions.md

```diff
@@ -119,6 +119,30 @@ export SERENITY_SOURCE_DIR=$(realpath ../)
 ./Build/ladybird/ladybird # or, in macOS: open ./Build/ladybird/ladybird.app
 ```
 
+### Debugging with Xcode on macOS
+
+The ``serenity.sh`` build script does not know how to generate Xcode projects, so creating the project must be done manually.
```

Comment:
Single backticks in markdown is the style we usually use

Reply:
Using single backticks for these new items now.